cluster-autoscaler/cloudprovider/hetzner/hetzner_manager.go

```diff
@@ -130,9 +127,20 @@ func newManager() (*hetznerManager, error) {
 		if err != nil {
 			return nil, fmt.Errorf("failed to unmarshal cluster config JSON: %s", err)
 		}
-	}
+		clusterConfig.IsUsingNewFormat = true
 
-	if !clusterConfig.IsUsingNewFormat {
+	} else if clusterConfigFile != "" {
+		clusterConfigData, err := os.ReadFile(clusterConfigFile)
+		if err != nil {
+			return nil, fmt.Errorf("failed to read cluster config file: %s", err)
+		}
+		err = json.Unmarshal(clusterConfigData, &clusterConfig)
+		if err != nil {
+			return nil, fmt.Errorf("failed to unmarshal cluster config JSON: %s", err)
+		}
+		clusterConfig.IsUsingNewFormat = true
```

Comment:
These lines are duplicated by the branches `clusterConfigBase64 != ""` and `clusterConfigFile != ""`. Could refactor it so they are shared?